lib/fathead/pip/fetch.py

```diff
@@ -0,0 +1,33 @@
+__author__ = 'kchahal'
```

Comment:
@karanchahal please add `#!/usr/bin/env python3` to the beginning of the `.py` files